src/DataFrames.jl

```diff
@@ -16,6 +16,8 @@ using Reexport
 using GZip
 using SortingAlgorithms
 
+using FileIO  # remove after read_rda deprecation period
```

Comment:
...but only is `RData` is loaded, right? So the dependency in REQUIRE should actually be `RData`? I was also a bit puzzled by the warning since it first points to `RData` and then mentions `FileIO`.